cmd/kubeadm/app/constants/constants.go

```diff
@@ -218,6 +218,9 @@ var (
 
 	// MinimumKubeletVersion specifies the minimum version of kubelet which kubeadm supports
 	MinimumKubeletVersion = version.MustParseSemantic("v1.8.0")
+
+	// MinimumUnstableVersion specifies the earliest version of the upcoming major release
+	MinimumUnstableVersion = version.MustParseSemantic("v1.9.0-alpha.1")
```

Comment:
MinimumPriorityAdmissionVersion would be a bit more descriptive IMO

Reply:
If we have v18 and v19 constants for admission controller list than I don't think this is needed.

Reply:
Well, yeah, either way.